### DISCUSSION TITLE
forge init

### DISCUSSION THREAD

**Discussion by cell0901** [Cyfrin/foundry-full-course-cu#708] [General]
whenever i use the "forge init" it adds all the files and folders except the lib/forge-std
and everytime i have to use this command to add lib/forge-std

``` 
git submodule add --force --name lib/forge-std https://github.com/foundry-rs/forge-std lib/forge-std
```

**Comment by theirrationalone** [comment]
Please make a `.gitmodules` file into your project at top level and stuffs you needed like this... [this](https://github.com/Cyfrin/foundry-fund-me-f23/blob/main/.gitmodules)

**Reply by cell0901** [reply]
i still don't understand, i already .gitmodules, but its different then one you mentioned
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/142923689/fc2b1e0d-1537-4143-8cbc-25e2351191d4)
where should i make it?

**Reply by theirrationalone** [reply]
If you Already have initialized project using forge init command and trying to reinitialize project again then you always need to do this `forge init --force`.

If you wanna initialize your repo then you have to use `git init`.

Please Make sure with your intentions whether you wanna initialize a repo Or wanna initialize a new project.
